### PR TITLE
[Maintenance] Upgrade JMS Serializer Bundle to 2.0

### LINKED ISSUE #8193: Upgrade JMS Serializer Bundle to 2.0
#8186 disabled aliasing `@serializer` service to `@jms_serializer.serializer` to increase compatibility with 3rd party addons for Sylius.

JMS Serializer Bundle in v2 disabled aliasing altogether, so probably the best course of action would be to upgrade the bundle to 2.0 instead of disabling the alias. However, it appears that Serializer Bundle 2.0 currently breaks Sylius, so this issue is opened as per @lchrusciel's instructions to track the progress.

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #8193  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/CustomerBundle/Resources/config/app/config.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@
 jms_serializer:
     metadata:
         directories:
-            sylius-user:
+            sylius-customer:
```
Comment (pamil) [inline src/Sylius/Bundle/CustomerBundle/Resources/config/app/config.yml]: 🥇

**Comment by lchrusciel** [comment]
Good catch @pamil

File: src/Sylius/Bundle/CustomerBundle/Resources/config/app/config.yml (hunk 1 above)
Comment (Zales0123) [reply src/Sylius/Bundle/CustomerBundle/Resources/config/app/config.yml]: 🎖

**Comment by pjedrzejewski** [comment]
Thanks Łukasz! 👍

**Pull request merged**